packages/opentelemetry-core/src/platform/node/index.ts

```diff
@@ -21,3 +21,4 @@ export * from './id';
 export * from './performance';
 export * from './sdk-info';
 export * from './timer-util';
+export * from './hex-to-base64';
```

Comment:
it might be merge conflicts but `hex-to-base64` is already included in line 19

Reply:
Thanks!